src/components/FloatingMenu/FloatingMenu.module.css

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.menu {
+  border-radius: var(--cpd-space-3x);
+  background: var(--cpd-color-bg-canvas-default);
+
+  /* Here we use outline to create a border internal to the container */
+  outline: var(--cpd-border-width-1) solid
+    var(--cpd-color-border-interactive-secondary);
+  outline-offset: calc(-1 * var(--cpd-border-width-1));
+  box-shadow: 0 4px 24px 0 rgba(0 0 0 / 10%);
```

Comment:
@janogarcia it looks like we're starting to use shadows like that in quite a few places. We were discussing making those tokens.
Should we pick this up again?

Reply:
That would be fantastic. In Element Call I ended up creating some CSS variables for them, but it would be even nicer if they could be formalized as tokens belonging to Compound.